### PR TITLE
DES-12693: adding data protocol to protocol list

### PR DESCRIPTION
**Pull request by rbarros-daitan**
Adding data protocol to allowed list fixes error on Selerity json export feature.

### CODE REVIEW

File: minuet/Paragon.Runtime/CefBrowserApplication.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ namespace Paragon.Runtime
 {
     public sealed class CefBrowserApplication : CefApp
     {
-        public static string[] AllowedProtocols = new string[] { "http", "https", "ws", "wss", "blob" };
+        public static string[] AllowedProtocols = new string[] { "http", "https", "ws", "wss", "blob", "data" };
```
Comment (lneir) [inline minuet/Paragon.Runtime/CefBrowserApplication.cs:25]: how does this fix issue in DES-12693?

Comment (rbarros-daitan) [reply minuet/Paragon.Runtime/CefBrowserApplication.cs:25]: Problem is that Selerity uses protocol "data" on JS to download json file. As this protocol is not allowed, error seen on screenshot included on DES-12693 occurs.

Comment (lneir) [reply minuet/Paragon.Runtime/CefBrowserApplication.cs:25]: what are the security implications?  is this allowed in chrome?

Comment (rbarros-daitan) [reply minuet/Paragon.Runtime/CefBrowserApplication.cs:25]: i Lynn, I could not find anything that indicates a security concern about enabling data protocol. It is largely used to access resources:
https://msdn.microsoft.com/pt-br/library/cc848897(v=vs.85).aspx

The other solution would be a change on Selerity side and asks them to use "blob" protocol to download json files.

Comment (lneir) [reply minuet/Paragon.Runtime/CefBrowserApplication.cs:25]: does chrome allow this protocol?

Comment (rbarros-daitan) [reply minuet/Paragon.Runtime/CefBrowserApplication.cs:25]: Yes, based on what is stated here:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

**Pull request merged**